climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -117,6 +121,12 @@ def fetch_ecmwf(self, path=None, files=None):
             except AttributeError:
                 pass
 
+            if os.name == 'nt':
+                try:
+                    file = file.file # if in windows try accessing the underlying tempfile directly incase variable file is tempfile._TemporaryFileWrapper
+                except AttributeError:
+                    pass
```

Comment:
I think we can avoid a try ... except here:
```suggestion
            if os.name == 'nt' and hasattr(file, 'file'):
                file = file.file # if in windows try accessing the underlying tempfile directly in case variable file is tempfile._TemporaryFileWrapper
```

Reply:
also, what is the code doing here?

Reply:
actually, this fits into a larger conversation about the module's file handling - we'll chat about it in person!

Reply:
update: we're moving this check inside `read_one_bufr_tc`